oam/install_scripts/run.sh

```diff
@@ -26,9 +29,9 @@ while getopts "vs:t:h" flag; do
 	esac
 done
 
-shift $((OPTIND - 1))
+shift $((OPTIND-1))
 
-exename="$1"
+exename="$@"
```

Comment:
This would make exename = all of the parameters instead of just the executable name